docs: adds link to Matrix in README (bug 1631971) (#104)
mozilla-conduit/lando-ui: README.md

[Auth0]: https://auth0.com/
[lando-api docker-compose.yml]: https://github.com/mozilla-conduit/lando-api/blob/master/docker-compose.yml
[Conduit Suite]: https://github.com/mozilla-conduit/suite

## Support

> To talk to the `lando` developers, you can join them on [Matrix](https://chat.mozilla.org/#/room/#lando:mozilla.org).

Reviewer: ```suggestion
To chat with Lando users and developers, join them on [Matrix](https://chat.mozilla.org/#/room/#conduit:mozilla.org).
```
